app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+    cp, file1, file2 = command.split()
+    if cp != "cp":
```

Comment:
Code Efficiency: You're checking if the command is 'cp', but you're doing it after you've already split the command and assigned the parts to variables. This means that if the command is not 'cp', you've done some unnecessary work. It would be more efficient to check if the command starts with 'cp' before splitting it.